src/Flex.php

```diff
@@ -39,13 +37,41 @@
  */
 class Flex implements PluginInterface, EventSubscriberInterface
 {
+    /**
+     * @var Composer
+     */
     private $composer;
+
+    /**
+     * @var IOInterface
+     */
     private $io;
+
+    /**
+     * @var Options
+     */
     private $options;
+
+    /**
+     * @var Configurator
+     */
     private $configurator;
+
+    /**
+     * @var Downloader
+     */
     private $downloader;
+
+    /**
+     * @var array
+     */
     private $postInstallOutput = [''];
+
+    /**
+     * @var \Closure|InstallCommand
```

Comment:
You're right, I didn't see that it was the return of this var that was type checked, not the var itself 👍